pkg/api/graph/graphview/dc_pipeline.go

```diff
@@ -12,8 +12,9 @@ import (
 type DeploymentConfigPipeline struct {
 	Deployment *deploygraph.DeploymentConfigNode
 
-	ActiveDeployment    *kubegraph.ReplicationControllerNode
-	InactiveDeployments []*kubegraph.ReplicationControllerNode
+	ActiveDeployment               *kubegraph.ReplicationControllerNode
+	ActiveDeploymentRestartingPods int32
```

Comment:
I still don't like that we have this field as part of the pipeline but if @deads2k and @smarterclayton who wrote this are ok with the change then I'm fine too